app/assets/javascripts/discourse/app/components/reviewable-user.gjs

```diff
@@ -1,89 +1,157 @@
 import Component from "@ember/component";
 import { concat } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
 import ReviewableField from "discourse/components/reviewable-field";
 import getUrl from "discourse/helpers/get-url";
+import { ajax } from "discourse/lib/ajax";
 import discourseComputed from "discourse/lib/decorators";
+import { REJECTED } from "discourse/models/reviewable";
 import { i18n } from "discourse-i18n";
+import ScrubRejectedUserModal from "admin/components/modal/scrub-rejected-user";
 
 export default class ReviewableUser extends Component {
+  @service currentUser;
+  @service modal;
+  @service store;
+
   @discourseComputed("reviewable.user_fields")
   userFields(fields) {
     return this.site.collectUserFields(fields);
   }
 
+  @discourseComputed("reviewable.status", "currentUser")
+  canScrubRejectedUser(status, currentUser) {
+    return status === REJECTED && currentUser.admin && !this.isScrubbed;
+  }
+
+  @discourseComputed("reviewable.payload")
+  isScrubbed(payload) {
+    return !!payload?.scrubbed_by;
+  }
+
+  @action
+  showScrubRejectedUserModal() {
+    this.modal.show(ScrubRejectedUserModal, {
+      model: {
+        confirmScrub: this.scrubRejectedUser.bind(this),
+      },
+    });
+  }
+
```

Comment:
You can use `@bind` decorator on top and skip manual bind `this.scrubRejectedUser.bind(this)`

Reply:
That's much neater, thanks!